conf/machine/include/stm32mp1-common.inc

```diff
@@ -1,5 +1,7 @@
 DEFAULTTUNE = "cortexa7thf-neon-vfpv4"
 
+MACHINEOVERRIDES =. "stm32mp1:"
```

Comment:
probably we should use SOC_FAMILY here [1].

[1] - https://www.yoctoproject.org/docs/latest/ref-manual/ref-manual.html#var-SOC_FAMILY